### PR TITLE
protocol: switch to binary MessageCommand

### LINKED ISSUE #888: Switch to binary MessageCommand
Neo 3 has no textual commands any more, so convert to binary codes.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
closes #888

### CODE REVIEW

File: pkg/network/server.go [hunk 1]
```diff
@@ -671,7 +671,7 @@ func (s *Server) requestBlocks(p Peer) error {
 func (s *Server) handleMessage(peer Peer, msg *Message) error {
 	s.log.Debug("got msg",
 		zap.Stringer("addr", peer.RemoteAddr()),
-		zap.String("type", string(msg.CommandType())))
+		zap.String("type", string(msg.Command)))
```
Comment (fyrchik) [inline pkg/network/server.go:674]: This is wrong way to do this, it converts `byte` to a byte in `utf-8` string.
Maybe implement `String()` method? (can be generated via `stringer`).

File: pkg/network/tcp_peer.go [hunk 2]
```diff
@@ -159,7 +159,7 @@ func (p *TCPPeer) handleConn() {
 			}
 			if err = p.server.handleMessage(p, msg); err != nil {
 				if p.Handshaked() {
-					err = fmt.Errorf("handling %s message: %v", msg.CommandType(), err)
```
Comment (fyrchik) [inline pkg/network/tcp_peer.go:162]: Same here, with `String()` method is will be a readable `%s`.

**Comment by roman-khimov** [comment]
Looks good, have you tested it with private net?

**Comment by AnnaShaleva** [comment]
> Looks good, have you tested it with private net?

Yes, consensus works with both #969 and #970

**Pull request merged**